src/packaging/Makefile

```
@@ -16,7 +16,7 @@
 package:
```

What if we also include the JAR path as a target?

```suggestion
REAPER_JAR=../server/target/cassandra-reaper-$(VERSION).jar

$(REAPER_JAR) package:
```